docs/en/latest/plugins/traffic-split.md

```diff
@@ -0,0 +1,178 @@
+---
+title: traffic-split
+---
+
+<!--
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+-->
+
+This guide describes how to use the traffic-split plugin in APISIX Ingress.
+
+The [traffic-split](https://apisix.apache.org/docs/apisix/plugins/traffic-split) plugin can be used to dynamically route part of traffic to various upstream services.
+
+:::info IMPORTANT
+
+If multiple services are declared in ApisixRoute `http[].backends`, it cannot be used with traffic-split.
+
+**Correct example:**
+
+```yaml
+
+backends:
+- serviceName: httpbin1
+  servicePort: 80 
+plugins:
+- name: traffic-split # It works.
+  enable: true
+
+```
+
+**Invalid examples:**
+
+```yaml
+backends:
+- serviceName: httpbin1
+  servicePort: 80 
+- serviceName: httpbin2
+  servicePort: 80
+plugins:
+- name: traffic-split # Not working, this plugin is invalid and cannot be used with multiple backend.
+  enable: true
+```
```

Comment:
I think it's better to give the full configuration instead of just partial snippets.